contrib/runners/action_chain_runner/action_chain_runner/runner.yaml

```diff
@@ -12,3 +12,10 @@
       default: []
       description: List of tasks to skip notifications for.
       type: array
+  output_schema:
```

Comment:
To be honest, I still think we should restrict where the action output schema gets applied to separately for each runner.  Having said that, for the sake of progressing this, I don't think `output_path` should be under `output_schema`, that's just confusing.  Also why is `output_path` a list?  I don't see any of the runners below having more than one output_path.

```
output_path: published
output_schema:
  published:
    type: object
  ...
```